app/models/energy/hvac.py

```diff
@@ -0,0 +1,45 @@
+"""HVAC Schema"""
```

Comment:
I think we should probably name this module `idealair.py` instead of `hvac.py`. Granted, so much stuff still needs to be figured out for us to support detailed HVAC but I know that we are definitely not going to put all HVAC-related schema stuff into a single file. Keeping just the ideal air system here is reasonable, though.